### DISCUSSION TITLE
Do you think we should allow everyone to create TApp in epoch3?

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#60] [Q&A] [answered]
In our current design, in epoch3, only sudo (the DAO simulator)can create a few youtube based TApps. The miners can invest using bonding curve, and use their social influence to pump the price. Do you think we should open to everyone to create TApp? or should we leave this feature to epoch 4?

**Comment by geopor** [comment, marked as answer]
It's probably not necessary, I don't think any of our contestants are YouTubers unless we invited them from YouTube ourselves.

One issue is we don't want the marketplace to be flooded with TApps.

An alternative method might be to ask them to list any social media channels when they sign up. If they have a YouTube channel, then we can invite them to create a YouTube TApp.
